cmd/initcmd/cmdinit.go

```diff
@@ -25,5 +25,6 @@ func NewCmdInit(ioStreams genericclioptions.IOStreams) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVarP(&io.InventoryID, "inventory-id", "i", "", "Identifier for group of applied resources. Must be composed of valid label characters.")
+	cmd.Flags().StringVarP(&io.Namespace, "inv-namespace", "", "", "namespace for the resources to be initialized")
```

Comment:
Let's keep the prefix on these identical, even if it means the namespace flag is a little bit long.

Reply:
Done.